pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_pantheon_yaml_file.adoc

```diff
@@ -17,17 +17,26 @@ The configuration file enables you to import content from a Git repository into
 The {ProductShortName} configuration file defines the following parameters:
 
 * `server` The URL of the {ProductShortName} server.
-* `repositories` A multi-valued structure with the following child members:
+* `repository`
+* `variants` A multi-valued structure with the following child members:
 +
 ** `name` (optional) A unique ID that displays in URLs on the {ProductShortName} site when you view modules and assemblies.
 +
 For example, if you specify `name: rhel_docs`, and one of your files is named [filename]`foo.adoc`, the module's URL in {ProductShortName} is:
 `\http://<_{ProductShortName}__url>/content/repositories/**rhel_docs**/entities/modules/getting_started/foo.adoc.preview`
```

Comment:
it should be:
`\http://<_{ProductShortName}__url>/content/repositories/**repository**/entities/modules/getting_started/foo.adoc?variant=rhel_docs`

"**repository**" is the value provided for `repository`